templates/partners/partial/_homepage-form.html

```diff
@@ -0,0 +1,89 @@
+<div class="p-modal" style="display: none;" id="executive-summit-modal">
+  <div class="p-modal__dialog" role="dialog" aria-labelledby="modal-title" aria-describedby="modal-description">
+    <header class="p-modal__header" style="display: block; border-bottom: 0;">
+      <button class="p-modal__close" aria-controls="executive-summit-modal" aria-label="Close active modal" style="margin-left: -1rem">Close</button>
+      <h2 class="p-modal__title u-sv1" id="modal-title">Contact Canonical</h2>
+    </header>
+    <div class="js-pagination js-pagination--1">
+      <div class="js-formfield">
+        <h3 class="p-heading--5">Please fill in this form and a member of our team will be in touch soon.</h3>
```

Comment:
I think the secondary heading should say: "_Fill in this form and we'll be in touch within one working day_." based on what's in the copy doc.